src/ProofOfConcepts/ACLFilterChips/TrasactionalIdLabel.js

```diff
@@ -10,9 +11,12 @@ class TransactionalIdLabel extends React.Component {
     return (
       <div class="row">
         <div class="child">
-          <span class="pf-c-label mas-m-solid smaller-label pf-m-orange ">
+          <Label
+            className="pf-c-label mas-m-solid smaller-label pf-m-orange"
+            isCompact
+          >
             <span class="pf-c-label__content">TI</span>
```

Comment:
When you replaced the first span with the `<Label>` you can also remove this span. This extra span is provided by the `<Label>` component for you.

Reply:
I applied both of the changes you mentioned :)